src/sentry/api/endpoints/release_thresholds/release_threshold_details.py

```diff
@@ -20,6 +21,8 @@
 from sentry.models.release_threshold.constants import TriggerType as ReleaseThresholdTriggerType
 from sentry.models.release_threshold.release_threshold import ReleaseThreshold
 
+logger = logging.getLogger("sentry.release_thresholds")
```

Comment:
Nope not yet
I had added some logs in, but figured if we're not looking for anything yet there's not really value.
Kept the logger just in case though